opengrok-tools/setup.py

```diff
@@ -0,0 +1,63 @@
+import os
+import unittest
+from setuptools import setup
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def readme():
+    with open(os.path.join(SCRIPT_DIR, 'src', 'main', 'python', 'README.md'),
+              'r') as readme:
+        return readme.read()
+
+
+def my_test_suite():
+    test_loader = unittest.TestLoader()
+    test_suite = test_loader.discover(
+        os.path.join(SCRIPT_DIR, 'src', 'test', 'python'), pattern='test_*.py')
+    return test_suite
+
+
+setup(
+    name='opengrok-tools',
+    version='1.1rc57',
```

Comment:
is there a way how to avoid hardcoding the version ? Should we be using OpenGrok version at all here actually ?

Reply:
We should and I think maven can do it, I hope someone how is with better experience with maven can handle this

Reply:
> >    think about something what to do with the python package version (replace with the current opengrok version?)

> yes please. That said, each major script has its own version.

You already said that we should use the opengrok version, that's why I changed it.